build.gradle

```diff
@@ -54,6 +53,50 @@ project.ext.set("pgdata", pgdata)
 project.ext.set("pgbindir", pgbindir)
 project.ext.set("targetroot", targetroot)
 
+// Define version variables for consistency
+project.ext.set("jacksonVersion", '2.15.2')
+
+def localFileDependencies = [
+        "home/WEB-INF/lib/agr_curation_api.jar",
+        "home/WEB-INF/lib/bbop.jar",
+        "home/WEB-INF/lib/biojava.1.7.1.jar",
+        "home/WEB-INF/lib/blast-serialization-1.0-eclipse-transformed.jar",
+        "home/WEB-INF/lib/commons-configuration-ant-task-0.9.7.jar",
+        "home/WEB-INF/lib/cvu.jar",
+        "home/WEB-INF/lib/gwt-servlet-jakarta-2.11.0.jar",
+        "home/WEB-INF/lib/jakarta.servlet.jsp.jstl-api-1.2.7-eclipse-transformed.jar",
+        "home/WEB-INF/lib/obo.jar",
+        "home/WEB-INF/lib/patricia-trie-0.2.jar",  // Not in Maven Central (0.6 version maybe "org.ardverk:patricia-trie:0.6",
+        "home/WEB-INF/lib/rescu-2.1.0-eclipse-transformed.jar",
+        "home/WEB-INF/lib/restygwt-2.2.7-eclipse-transformed.jar",
+        "home/WEB-INF/lib/taglibs-standard-impl-1.2.5-eclipse-transformed.jar",
+        "home/WEB-INF/lib/text-table-formatter-1.0.jar",
```

Comment:
Can we use this? https://mvnrepository.com/artifact/org.nocrala.tools.texttablefmt/text-table-formatter/1.2.4